core/embed/rust/src/ui/model_mercury/component/number_input_slider.rs

```diff
@@ -135,20 +129,27 @@ impl NumberInputSlider {
         Self {
             area: Rect::zero(),
             touch_area: Rect::zero(),
+            text_area: Rect::zero(),
             min,
             max,
             value,
+            touching: false,
         }
     }
 
-    pub fn slider_eval(&mut self, pos: Point, ctx: &mut EventCtx) -> Option<u16> {
+    pub fn touch_eval(
+        &mut self,
+        pos: Point,
+        ctx: &mut EventCtx,
+        force_bubble_up: bool,
+    ) -> Option<u16> {
         if self.touch_area.contains(pos) {
```

Comment:
I wonder if this condition should be replaced with `self.touching`. This would make it possible to change the value even when finger slides oudside rectangle, provided the TouchStart happened inside. Thinking of situation where you notice the footer changed and move your finger to read it, fat fingers, etc ...

Reply:
Indeed, it feels better ... https://github.com/trezor/trezor-firmware/pull/3991/commits/cf5be6ff82a54b5abf521a1f9877cc5bc9069663#diff-6e51ef35492aeba10734987923ac867ce64c62e70b23c8a78e47ee86c122c4b6R146